tasks/docker.yml

```diff
@@ -43,7 +43,7 @@
 
 - name: Ensure pi user is added to the docker group.
   ansible.builtin.user:
-    name: pi
+    name: "{{ ansible_user }}"
```

Comment:
Please open a separate PR for this fix — and also make the same change up in the `name` above (where it says `pi`).